src/main/java/org/mule/extension/objectstore/api/ExtensionObjectStore.java

```diff
@@ -156,6 +156,11 @@ public void start() throws MuleException {
     storeManagerProvider = getObjectStoreManagerProvider();
     objectStoreManager = getObjectStoreManager();
 
+    if (maxEntries < 0) {
+      LOGGER
+          .warn("The maxEntries parameter should not be negative, otherwise there is no guarantee that the expiration policy will work correctly. For unlimited entries, just omit passing this parameter");
```

Comment:
I would add to the message which is the value that maxEntries has.

Reply:
Done